### ISSUE TITLE
Multifactor Code Requested On Subsequent Runs Even With Session Path

### ISSUE THREAD

**Issue by pisakhov** [mintapi/mintapi#419]
Hey Everyone!

Ubuntu 20.04.4 LTS (GNU/Linux 5.13.0-1017-aws x86_64)
Python 3.8.10
ChromeDriver 99.0.4844.51 (d537ec02474b5afe23684e7963d538896c63ac77-refs/branch-heads/4844@{#875})
mintapi                  1.64

`mint = mintapi.Mint(
    '####@#####.com',  
    '########',
    mfa_method='sms',  
    mfa_input_callback=None,
    mfa_token=None,
    intuit_account=None,
    headless=True,
    session_path='/home/ubuntu/.mintapi/session',
    #imap_account=None, # account name used to log in to your IMAP server
    #imap_password=None, # account password used to log in to your IMAP server
    #imap_server=None,  # IMAP server host name
    #imap_folder='INBOX',  # IMAP folder that receives MFA email
    #wait_for_sync=False,  # do not wait for accounts to sync
    #wait_for_sync_timeout=300,  # number of seconds to wait for sync
	use_chromedriver_on_path=False,
)`

It sends me a verification code to my email I enter but never saves it.
It worked before I updated to mintapi 1.64
I tried all mfa_methods, different folders to save, and nothing works.

Thank you!

**Comment by pisakhov** [comment]
![image](https://user-images.githubusercontent.com/64985264/162595385-9f321c7f-83c1-49ee-a4b2-fa76e5125711.png)

I turned off headless and tried to find out more about this problem.

It saves the session, but instead of going to the account I logged in to before (xxxx@gmail.com), it goes to the 'Use a different user ID'.

Note: I added to intuit_account='' and it still goes to the 'Use a different user ID' option.

Thank you!
